core/application/impl/app_configuration_impl.cpp

```diff
@@ -315,6 +318,7 @@ namespace kagome::application {
         ("single_finalizing_node,f", "if this is the only finalizing node")
         ("already_synchronized,s", "if need to consider synchronized")
         ("unix_slots,u", "if slots are calculated from unix epoch")
+        ("dev", "if node run in development mode")
```

Comment:
I think it would be great to shortly explain here what that means.

Reply:
Right now it is just allowing to run node without any bootnodes. In future in might be used for additional things. I guess it should be explained in documentation, if really needed.